src/application/components/Explorer/Explorer.js

```diff
@@ -32,7 +32,6 @@ export const createBridgeLink = bridge =>
   new ApolloLink(
     operation =>
       new Observable(obs => {
-        const key = operation.toKey();
```

Comment:
`toKey` is removed from ApolloLink. I started seeing errors when trying to run the application so this is now (temporarily) replaced with the `operationName`.